virt/about-virt.adoc

```diff
@@ -11,6 +11,7 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+1]
 
 // This line is attached to the above `virt-what-you-can-do-with-virt` module.
 // It is included here in the assembly because of the xref ban.
-You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+
+You can use {VirtProductName} with the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes], xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN], or one of the other certified default Container Network Interface (CNI) network providers listed in link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] that were confirmed to pass with {VirtProductName}.
```

Comment:
"that were confirmed to pass with {VirtProductName}" makes the sentence pretty long, is that necessary or could it be removed? You already say that they are certified earlier in the sentence.

If you do keep it, "confirmed to pass" sounds a little awkward. I'd try something else. But I do think that you could just leave that whole part off still.

Reply:
Sure thing. Let me trim that!

Reply:
Trimmed away!